src/styles/utils/variables.scss

```
@@ -0,0 +1,12 @@
+$daySize: 100px;
+$gapSize: 1px;
+
+$days:(
+  "Monday": 0,
+  "Tuesday": 1,
+  "Wednesday": 2,
+  "Thursday": 3,
+  "Friday": 4,
+  "Saturday": 5,
+  "Sunday": 6,
```

This doesn't work. Your selector in index.html is --sun , it  should be "sun" here as well
```suggestion
  "mon": 0,
  "tue": 1,
  "wed": 2,
  "thu": 3,
  "fri": 4,
  "sat": 5,
  "sun": 6,
```